server/routes/users.js

```diff
@@ -300,12 +300,16 @@ export default (storage, scriptManager) => {
                 return accessToken;
               }))
           .then((accessToken) => {
-            if (data.user.multifactor && data.user.multifactor.indexOf('guardian') >= 0) {
+            const checkGuardian = !!data.user.multifactor_last_modified
+              || (data.user.multifactor && data.user.multifactor.indexOf('guardian') >= 0);
+            if (checkGuardian) {
               return requestGuardianEnrollments(accessToken, req.params.id)
                 .then((enrollments) => {
-                  if (!enrollments || !enrollments.length) {
+                  if (data.user.multifactor && (!enrollments || !enrollments.length)) {
                     data.user.multifactor = data.user.multifactor.filter(item => item !== 'guardian');
                     data.user.multifactor = data.user.multifactor.length ? data.user.multifactor : null;
+                  } else if (enrollments && !data.user.multifactor) {
```

Comment:
can we keep the condition ordering consistent with [L308](https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/178/files#diff-e248e0ee41a5c0360b4f480728bc0e8bR308)?

`else if (!data.user.multifactor && enrollments) {`

Reply:
Done.